modules/services/cache/main.tf

```diff
@@ -0,0 +1,46 @@
+resource aws_elasticache_cluster redis {
+  cluster_id = var.redis_cluster_id
+  engine = "redis"
+  node_type = var.redis_node_type
+  num_cache_nodes = var.redis_num_cache_nodes
```

Comment:
Huh.. you said redis doesn't support multiple nodes? If not, then this should be hardcoded with a comment.

Reply:
thank you for spotting this :+1: